Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -577,6 +596,7 @@
                                   Grid.Column="0" 
                                   Grid.ColumnSpan="2"
                                   x:Name="PART_MainContent"
+                                  Height="{TemplateBinding Height}"
```

Comment:
This looks wrong to me. What if the parent's `Height` is not set? It's probably a better idea to handle layout sizing in code behind.

Reply:
Thanks for the feedback! I removed that line and worked.